guardian/prosemirror-elements
# fix: Update yarn.lock

## What does this change?

Updates yarn.lock, as multiple versions of `prosemirror-model` were causing problems with the demo editor.

Before:

```
15:37 $ yarn list prosemirror-model
yarn list v1.22.10
warning Filtering by arguments is deprecated. Please use the pattern option instead.
├─ prosemirror-commands@1.1.10
│  └─ prosemirror-model@1.13.3
├─ prosemirror-dropcursor@1.3.3
│  └─ prosemirror-model@1.13.3
├─ prosemirror-example-setup@1.1.2
│  └─ prosemirror-model@1.13.3
├─ prosemirror-gapcursor@1.1.5
│  └─ prosemirror-model@1.13.3
├─ prosemirror-history@1.1.3
│  └─ prosemirror-model@1.13.3
├─ prosemirror-inputrules@1.1.3
│  └─ prosemirror-model@1.13.3
├─ prosemirror-menu@1.1.4
│  └─ prosemirror-model@1.13.3
├─ prosemirror-model@1.14.2
├─ prosemirror-schema-basic@1.1.2
│  └─ prosemirror-model@1.13.3
├─ prosemirror-schema-list@1.1.4
│  └─ prosemirror-model@1.13.3
├─ prosemirror-state@1.3.4
│  └─ prosemirror-model@1.13.3
├─ prosemirror-transform@1.3.2
│  └─ prosemirror-model@1.13.3
└─ prosemirror-view@1.18.9
   └─ prosemirror-model@1.13.3
✨  Done in 0.46s.
```

After:

```
15:49 $ yarn list prosemirror-model
yarn list v1.22.10
warning Filtering by arguments is deprecated. Please use the pattern option instead.
└─ prosemirror-model@1.14.3
✨  Done in 0.47s.
```

This was causing a problem where hitting enter at the end of a document caused an error rather than updating the document.

## How to test

Run `yarn`. Add a newline at the end of a document. Does the operation work as expected?
